src/store/MenuStore.js

```diff
@@ -31,9 +31,7 @@ export default class MenuStore {
     }
 
     @action.bound onTitleClick(e) {
-        /* TODO: why stopPropagation() is not working ಠ_ಠ */
-        // e.stopPropagation();
-        e.nativeEvent.isHandledByDialog = true;
+        e.stopPropagation();
```

Comment:
@aminroosta now that we use a native event instead of React's event, this should work.